tipg/settings.py

```diff
@@ -86,6 +86,7 @@ class FeaturesSettings(BaseSettings):
 
     default_features_limit: int = Field(10, ge=0)
     max_features_per_query: int = Field(10000, ge=0)
+    sort_columns: bool = True
```

Comment:
I wonder if we should have this set to `False` by default for the users that already use tipg 

I don't think it will hurt that much to be honest but 🤷 

Maybe we should sort the column in postgres directly 

cc @bitner

Reply:
@vincentsarago - My sentiment was to do everything to leave current tipg behavior intact (I hate breaking stuff for established users), but if this were greenfield dev, I'd probably have turned sorting *off* by default.